man/man8/zfs.8

```diff
@@ -1329,6 +1334,34 @@ are equivalent to the
 and
 .Sy nodev
 mount options.
+.It Sy dedup Ns = Ns Sy off Ns | Ns Sy on Ns | Ns Sy verify Ns | Ns Sy \
+sha256[,verify] Ns | Ns Sy sha512[,verify] Ns | Ns Sy skein[,verify] Ns | Ns \
```

Comment:
nit: i don't think we really need these two "\\" here, see "dnodesize" and "encryption" properties on how they deal with multiline.